src/animad-keepers-view.html

```diff
@@ -23,7 +24,17 @@
       }
     </style>
 
-    <div class="card">
+    <app-route route="{{route}}" pattern="/:page" data="{{routeData}}" tail="{{subroute}}">
+    </app-route>
+
+    <iron-pages selected="[[page]]" attr-for-selected="name" fallback-selection="view404">
+        <animad-keepers-list name="" url="http://localhost:39146/keepers" default-search='vorname:"hans"'></animad-keepers-list>
+<!--        <animad-keepers-list name="list" url="http://localhost:39146/keepers" default-search='vorname:"hans"'></animad-keepers-list>-->
+        <animad-keeper-create-form name="create" url="http://demo6512140.mockable.io/keepers/new"></animad-keeper-create-form>
+        <animad-keeper-update-form name="edit" load-url="http://localhost:39146/keepers{{subroute.path}}"></animad-keeper-update-form>
```

Comment:
Woher kommt hier dieses {{subroute}}? An die Stelle muss ja die ID des Objektes. Mir ist nicht klar woher die im view Kontext kommt.

Reply:
Eigentlich das gleiche wie unter dem vorigen Kommentar. Weitergeleitet wird "/keepers-view/keepers/{item.id}". In animad-keepers-view ist die subroute dann {item.id}

Reply:
Das ist genau mein Problem. Ich habe eine Seite mit einer Suche, in der ich "Keepers" suchen kann. Warum hat die eine "item.id" in der URL? Das habe ich noch nicht verstanden.

Reply:
Beim Edit setze ich windows.location (also in Zukunft route.path ) auf /keepers-view/keepers/'die-id-des-items-der-angeklickten-zeile'. Auf animad-app ist subroute.path (gesetzt über den teil der route) noch /keepers/'die-id...', in animad-keepers-view ist der subroute.path die id selbst, da hier keepers die page ist.

Reply:
Ich glaub ich hab es immer noch nicht 100 Prozent kapiert. Schau in mir an :)